cmd/medius/images/push.go

```diff
@@ -261,8 +298,22 @@ func (b *buildAndPublish) pushImage(containerDisk v1.Image, name string) error {
 	return nil
 }
 
+func (b *buildAndPublish) pushImageIndex(containerDiskIndex v1.ImageIndex, name string) error {
+	if !b.Options.DryRun {
```

Comment:
This code pattern could use a cleanup to use early returns, but let's do that in a follow up.